src/app/components/LiveLabel/index.tsx

```diff
@@ -50,15 +51,24 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    styles.pulseContainer,
+    className === 'first-promo' && styles.firstPromo,
   ];
```

Comment:
And as we discovered today (topic pages 404ing 😬 ), this can sometimes be bad (although I am assuming emotion can handle it OK): https://bbc-tpg.slack.com/archives/C03RK72084S/p1701872689679539?thread_ts=1701870711.804219&cid=C03RK72084S

Should we do this, just to be safe, or is it overkill?

```suggestion
const circle = [
    styles.pulseContainer,
    className === 'first-promo' && styles.firstPromo,
  ].filter(Boolean);
```

Reply:
I think its ok to leave as is, Emotion can handle falsy values in their `css` function: https://emotion.sh/docs/@emotion/css#array-of-object-styles